### PR TITLE
Mongoid Paranoia Sync

### PR DESCRIPTION
**Pull request by jwg2s**
I'm adding some aliases to scopes so that the methods sync with Mongoid Paranoia methods. We're using polygot databases and I think it's nicer to have consistent methods across components tasked with doing the same thing.

Maybe one day I'll combine the two gems to work with ActiveRecord & Mongoid.

Also contains formatting changes.  If you use different formatting settings with your IDE I can adjust.

https://github.com/simi/mongoid-paranoia

### CODE REVIEW

File: lib/paranoia.rb [hunk 1]
```diff
@@ -4,11 +4,14 @@ def self.included(klazz)
...
 
   module Query
-    def paranoid? ; true ; end
+    def paranoid?
+      true
+    end
 
     def only_deleted
-      scoped.tap { |x| x.default_scoped = false }.where("#{self.table_name}.deleted_at is not null")
+      scoped.tap { |x| x.default_scoped = false }.where("#{self.table_name}.deleted_at IS NOT NULL")
```
Comment (radar) [inline lib/paranoia.rb:12]: There is no difference here ;)

File: lib/paranoia.rb [hunk 2]
```diff
@@ -44,13 +49,18 @@ def update_attribute_or_column(*args)
 class ActiveRecord::Base
   def self.acts_as_paranoid
     alias :destroy! :destroy
-    alias :delete!  :delete
+    alias :delete! :delete
     include Paranoia
-    default_scope { where(:deleted_at => nil) }
+    default_scope { where(deleted_at: nil) }
```
Comment (radar) [inline lib/paranoia.rb:54]: Possible potential to break Ruby 1.8 compatibility. I like it.

File: lib/paranoia/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module Paranoia
-  VERSION = "1.3.1"
+  VERSION = '1.3.1'
```
Comment (radar) [inline lib/paranoia/version.rb:2]: webscale!

File: paranoia.gemspec [hunk 4]
```diff
@@ -1,27 +1,27 @@
 # -*- encoding: utf-8 -*-
-require File.expand_path("../lib/paranoia/version", __FILE__)
+require File.expand_path('../lib/paranoia/version', __FILE__)
```
Comment (radar) [inline paranoia.gemspec:2]: more webscale!

File: paranoia.gemspec [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
-  s.email       = []
-  s.homepage    = "http://rubygems.org/gems/paranoia"
-  s.summary     = "Paranoia is a re-implementation of acts_as_paranoid for Rails 3, using much, much, much less code."
+  s.name = 'paranoia'
+  s.version = Paranoia::VERSION
+  s.platform = Gem::Platform::RUBY
+  s.authors = %w(radarlistener@gmail.com)
+  s.email = []
+  s.homepage = 'http://rubygems.org/gems/paranoia'
+  s.summary = 'Paranoia is a re-implementation of acts_as_paranoid for Rails 3, using much, much, much less code.'
```
Comment (radar) [inline paranoia.gemspec:11]: even more webscale!

File: test/paranoia_test.rb [hunk 6]
```diff
@@ -1,6 +1,7 @@
 require 'test/unit'
 require 'active_record'
-require File.expand_path(File.dirname(__FILE__) + "/../lib/paranoia")
+require 'debugger'
```
Comment (radar) [inline test/paranoia_test.rb]: Please remove this line. This is not webscale at all.

**Comment by radar** [comment]
Please remove the require for debugger and then this patch is good to go.

File: lib/paranoia.rb (hunk 1 above)
Comment (jwg2s) [inline lib/paranoia.rb:12]: I know, I was just taught to put certain SQL phrases in all caps, just a style thing -- I can undo if you'd like

File: test/paranoia_test.rb (hunk 6 above)
Comment (jwg2s) [inline test/paranoia_test.rb]: Ah, you caught me.

File: lib/paranoia.rb (hunk 2 above)
Comment (jwg2s) [inline lib/paranoia.rb:54]: Out with the old

**Comment by jwg2s** [comment]
Should be good to go

File: lib/paranoia.rb (hunk 1 above)
Comment (radar) [inline lib/paranoia.rb:12]: I am pedantic but not that pedantic. Leave it be :)

**Comment by radar** [comment]
Merged, thanks!

**Pull request merged**